tests/users/test_api.py

```diff
@@ -295,3 +292,16 @@ async def test_remove_permission(spawn_client, role, status, snapshot):
 
     assert resp.status == status
     assert await resp.json() == snapshot()
+
+
+async def test_first_user_view(spawn_client):
+    """
+    Checks response when there already is a user.
+    """
+    client = await spawn_client(authorize=True, administrator=True)
+
+    resp = await client.put(
+        "/users/first", {"handle": "fred", "password": "hello_world"}
+    )
+
+    assert resp.status == 409
```

Comment:
I think this test should cover two cases:

 1. That no users exist, and a new user is created with the passed info
 2. That a user does exist and no user is created

Since `spawn_client` creates a user, this test is only checking case 2.